digest_generator.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
                           to be compatible with postgres bigints, supports xxHash32 and xxHash64 also'
   spec.homepage      = 'https://github.com/YourMechanic/digest_generator'
   spec.license       = 'MIT'
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.6.2')
+  # spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
```

Comment:
any reason this is commented out?

Reply:
@siddyg required ruby version 2.3.0 has rubocop warning we saw in slack. So I removed required ruby version and documented the rubocop in todo file.